### ISSUE TITLE
Insufficient contrast ratio

### ISSUE THREAD

**Issue by cyntss** [EOS-uiux-Solutions/eos-icons-landing#11]
Importing issues from old repo on Gitlab.
Created by: @Aman-Codes

-----

Background and foreground colors should have a sufficient contrast ratio.

![image](https://user-images.githubusercontent.com/5013096/136550991-a4f57d54-5b2a-494a-bcac-2ad1aa3be19c.png)

**Comment by AnshumanDhiman** [comment]
can I work on this issue ?

**Comment by AnshumanDhiman** [comment]
I reviewed the website, what I found that maybe the contrast ratio is fixed.
The contrast ratio currently is lagging in the tags which are mentioned when a user click on an icon
![issue_1_prob_coming_where](https://user-images.githubusercontent.com/67755381/154796549-e71ae47e-47b4-40da-b298-64aee82c3d81.jpg)
![issue_1_icons_select_tags_coming](https://user-images.githubusercontent.com/67755381/154796566-c78095c7-a4af-4ab2-bbae-1fc37eaefc3f.jpg)
We can add a bit dark contrast to the tags so that the contrast ratio can be maintained.

Another thing I found out was the headings and the text both have a font weight which is lighter, due to which they both are sub-merging in each other.
We can set it from lighter to bolder, so that It can improve the UI design.

This is an example of the current density of headings in the page
![current](https://user-images.githubusercontent.com/67755381/154796672-8aa60465-12c4-421a-89d8-4cf1eb9d5d8f.jpg)

This is an example showing the headings bolder.
![bolder](https://user-images.githubusercontent.com/67755381/154796671-f2a51a85-fc00-43b5-ab3e-b0843b1ebcc0.jpg)

This is an example showing the headings bold.(This is creating conflict as the subheading and heading both are bold.)
![bold](https://user-images.githubusercontent.com/67755381/154796666-da81becf-0cb8-4bd1-b36f-436fd68a77b9.jpg)

**Comment by cyntss** [comment]
Ok with the fix in contrast for labels!

as for the headings, I would go for the least disruptive options for now:
![image](https://user-images.githubusercontent.com/5013096/154941376-2184c764-18bd-4718-acd0-cf52543d0302.png)
